src/classes/LineItemTriggerHandler.cls

```diff
@@ -12,6 +12,7 @@ public with sharing class LineItemTriggerHandler {
 		Set<Id> orderIds = new Set<Id>();
 		for (Line_Item__c i : items) {
 			orderIds.add(i.Order__c);
+            update i;
```

Comment:
This data manipulation is inside a loop. Please use a collection to store relevant data, and move any database manipulation outside the loop.